README.md

```diff
@@ -1 +1,613 @@
-# rasaxctl
\ No newline at end of file
+# rasactl
+
+rasactl deploys Rasa X / Enterprise on your local or remote Kubernetes cluster and manages Rasa X / Enterprise deployments.
+
+## Features
+
+- deploy Rasa X / Enterprise
+- upgrade Rasa X / Enterprise
+- stop/delete/start a running Rasa X / Enterprise deployment
+- connect a local Rasa Server to Rasa X / Enterprise
+- mount a local Rasa project to Rasa X / Enterprise deployment
```

Comment:
Kinda, with Rasa X 1.0 the local mode that we know in the current form will be removed, instead of it you can run a Rasa X deployment locally that uses your local Rasa project/directory. 

The thing that happens underneath is that a local directory is mounted to a Rasa X deployment. I think maybe it'll be better to say something like "use a local Rasa project directory"?